Controller/FolderController.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * This file is part of the ezsystems/demo-bundle package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\DemoBundle\Controller;
+
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use eZ\Bundle\EzPublishCoreBundle\Controller;
+use eZ\Publish\Core\Pagination\Pagerfanta\ContentSearchAdapter;
+use Pagerfanta\Pagerfanta;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+use eZ\Publish\API\Repository\Values\Content\Location;
+
+class FolderController extends Controller
+{
+    /**
+     * Displays the sub folder if it exists
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\Location $location of a folder
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function showFolderListAsideViewAction( Location $location )
```

Comment:
If I recall correctly, this controller wasn't referenced anywhere.

What is it called by ?